test/object-store/CMakeLists.txt

```diff
@@ -93,7 +93,32 @@ endif()
 target_link_libraries(ObjectStoreTests Catch2::Catch2 ObjectStore RealmFFIStatic)
 enable_stdfilesystem(ObjectStoreTests)
 create_coverage_target(generate-coverage ObjectStoreTests)
-add_bundled_test(ObjectStoreTests)
+
+# add_bundled_test(ObjectStoreTests)
+add_labeled_test(ObjectStoreTests-local objstore-local ObjectStoreTests "~[baas]")
+
+# Baas server and custom tests are only allowed when REALM_ENABLE_AUTH_TESTS is set
+if(REALM_ENABLE_SYNC)
+    if(REALM_ENABLE_AUTH_TESTS)
+        set(OBJSTORE_TESTS_FILE "${CMAKE_CURRENT_SOURCE_DIR}/object-store-tests.lst")
```

Comment:
you are not supposed to ever commit this file, right? It should be enforced somehow.

Reply:
Unfortunately, there isn't an easy way to do this - most of the recommendations are in the local repo (`git update-index --assume-unchanged <file>`) or using `.gitignore`, which requires deleting the file. An alternative way would be to create a pre-receive hook in Github, which would reject the push if the changes include the modified file, but this would require comparing git versions (new vs. old) on every push to see if that file was in there...

I could add a statement to the file and CMakeLists.txt that mentions "DO NOT COMMIT"...

Reply:
could you may be just then rename this file to something like "DO_NOT_COMMIT_object_store_tests.lst" as an additional reminder?